atom/renderer/api/atom_api_web_frame.cc

```diff
@@ -188,15 +188,14 @@ void WebFrame::DetachGuest(int id) {
 
 void WebFrame::SetSpellCheckProvider(mate::Arguments* args,
                                      const std::string& language,
-                                     bool auto_spell_correct_turned_on,
```

Comment:
Is this a breaking change? Our app currently has this code which seems like it will now fail:

`webFrame.setSpellCheckProvider(locale, true, provider);`